gammapy/modeling/models/tests/test_io.py

```diff
@@ -124,6 +124,17 @@ def test_sky_models_io(tmpdir, models):
     # or check serialised dict content
 
 
+@requires_data()
+def test_sky_models_checksum(tmpdir, models):
+    models.write(
+        tmpdir / "tmp.yaml", full_output=True, overwrite_templates=False, checksum=True
+    )
+    file = open(tmpdir / "tmp.yaml", "rb")
+    yaml_content = file.read()
+
+    assert "checksum: " in str(yaml_content)
```

Comment:
Maybe you can add a test on the checksum it-self. IN this way, we are sure that the checksum computation is stable over time.